app/src/main/res/values/themes.xml

```diff
@@ -1,17 +1,16 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
     <!-- Base application theme. -->
-    <style name="Theme.Sharingang" parent="Theme.MaterialComponents.DayNight.NoActionBar">
+    <style name="Theme.Sharingang" parent="Theme.Design.NoActionBar">
```

Comment:
Could we keep the Material design theme and change the colors?
The buttons are a bit bigger and there is an animation when clicking on it